.github/workflows/build-and-test.yml

```diff
@@ -103,17 +103,46 @@ jobs:
                     requirements-gui.txt > requirements-all.yml
                   conda env update --file requirements-all.yml
 
+            - name: Conda Environment Listing
+              shell: bash -l {0}
+              run: |
+                  conda list
+                  conda list --export > conda-env.txt
```

Comment:
Is this double `conda list` intentional?

Reply:
Yes! to "print" for viewing in Github Actions to see the full conda environment being used for the build and tests. And the second one to save to a file to compare against and archive later.